### PR TITLE
Implement MoneyDoor plugin.

### PR DESCRIPTION
**Pull request by SuboOx**
This might not be a perfect design.

### CODE REVIEW

**Comment by EmperorRay** [comment]
You suxx.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
+
+class MoneyDoorPlugin implements GamePlugin {
```
Comment (EmperorRay) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java:9]: hahahahahahaha

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
```
Comment (EmperorRay) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java:11]: sadfasdfasdfsadf

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java (hunk 1 above)
Comment (SuboOx) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java:9]: Hohohohoho

**Review by EmperorRay — APPROVED** [review]

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java (hunk 1 above)
Comment (EmperorRay) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoor.java:9]: good

**Pull request merged**